IQDropDownTextField/IQDropDownTextField.h

```diff
@@ -143,6 +143,22 @@ extern NSInteger const IQOptionalTextFieldIndex;
  */
 @property(nullable, nonatomic,copy)   NSAttributedString     *attributedText NS_UNAVAILABLE;
 
+/**
+ Sets a custom font for the IQDropdownTextField items. Default is boldSystemFontOfSize:18.0.
+ */
+@property (nullable, strong, nonatomic) UIFont *font;
```

Comment:
Looks like `font` & `textColor` property already exist with UITextField, but we can use `optionalItemTextColor`. However unintentionally you've encouraged me to use UITextField property to render texts in within UIPickerView. Thanks for this 👍